README.md

```diff
@@ -1,31 +1,141 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-# Linters Test
-> This project is to test linters
-## Built With
-- HTML and CSS
+<a name="readme-top"></a>
 
-## Getting Started
+<div align="center">
+
+  <img src="kefi-Unearthed-5" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] The link to your logo is broken. Kindly fix it so that your _**README.md**_ file looks professional. 👍

Reply:
Thank you for your change request. The README has been corrected.

Reply:
Well resolved!